bitbots_docs/docs/manual/hardware/bitfoot.rst

```diff
@@ -48,48 +37,50 @@ Next we connect some wires:
 Connect the USB to serial converter to your computer using a Micro USB cable.
 
 
-Run the install script in the Arduino_STM32 folder
+Run the install script in the ``Arduino_STM32`` folder
 
 :code:`./Arduino_STM32/tools/linux64/install.sh`
 
-Then you need to logout and back in again and remove and replug the USB to serial device.
+Logout and back in again.
+Then remove and replug the USB to serial device.
 
-Locate the script for flashing the bootloader in the Arduino_STM32 directory:
+Locate the script for flashing the bootloader in the ``Arduino_STM32`` directory:
 
 :code:`Arduino_STM32/tools/linux64/stm32flash`
 
 Run the flash script:
 
 :code:`./stm32flash -w <path to generic_boot20_pc13.bin> -v /dev/ttyUSB0 -b 115200`
 
-When the bootloader is successfully installed, remove the USB to serial converter and its connections to the STM32 board,
+When the bootloader is successfully installed, remove the USB to serial converter and its connections to the STM32 board.
 Then place the boot 0 pin back to 0.
 
-When connecting it to a computer using a micro usb cable it should appear as a ttyACM0 device in the /dev/ folder.
+When connecting it to a computer using a Micro-USB cable, it should appear as a ``ttyACM0`` device in the ``/dev/`` folder.
 
-Clone the repository_ and open bit_foot/firmware/main/main.ino in the Arduino IDE.
+Clone the `repository <https://www.github.com/bit-bots/bit_foot>`_ and open ``bit_foot/firmware/main/main.ino`` in the Arduino IDE.
 
-Since the Arduino_STM32 folder should already be in your Arduino directory, the board should already be installed.
+Since the ``Arduino_STM32`` folder should already be in your Arduino directory, the board should already be installed.
 
-You still need to install the Arduino SAM boards from the Board Manager to install the correct compiler.
+You still need to install the ``Arduino SAM`` boards from the ``Board Manager`` to install the correct compiler.
 
 Before compiling select:
 
-* Board: Generic STM32F103C series
-* Variant: STM32F103C8 20k RAM. 64k Flash (The other variant may also work)
-* CPU Speed: 72 Mhz
-* Upload Method: STM32duino Bootloader
+* Board: ``Generic STM32F103C series``
+* Variant: ``STM32F103C8 20k RAM 64k Flash`` (The other variants may also work)
+* CPU Speed: ``72 Mhz``
+* Upload Method: ``STM32duino Bootloader``
 * Optimize: not clear yet... (use default for now)
```

Comment:
@jgueldenstein Still unclear?

Reply:
leave it for now, it is outdated and the new board uses an esp32 anyways